### PR TITLE
Fixes typographical error when group member is removed from a group. 

### LINKED ISSUE #1227: Report User Successfully Removed instead of User Successfully Destroyed.
**Describe the bug**
When removing a member from the group, a message is displayed stating "Group member was successfully _destroyed_" which is awkward.

**To Reproduce**
1. Go to 'http:circuitverse.org'
2. Click to see your groups if you are a mentor.
3. Remove any member.

**Expected behavior**
Group member was successfully removed.

**Screenshots**
Screenshot from 2020-03-18 12-27-41

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Browser: Chrome

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by ShubhankarKG**
Fixes #1227 

#### Describe the changes you have made in this PR -
Replaced Group member destroyed with Group member removed.

### CODE REVIEW

**Comment by ShubhankarKG** [comment]
@Shivansh2407  @nitin10s please review .

**Comment by ShubhankarKG** [comment]
Screenshot from 2020-03-18 16-58-39

Changes done.

File: app/controllers/group_members_controller.rb [hunk 1]
```diff
@@ -90,10 +90,10 @@ def create
   # DELETE /group_members/1
   # DELETE /group_members/1.json
   def destroy
-
+    noticeMessage = "Group member was successfully removed."
     @group_member.destroy
     respond_to do |format|
-      format.html { redirect_to group_path(@group_member.group), notice: 'Group member was successfully destroyed.' }
+      format.html { redirect_to group_path(@group_member.group), notice: noticeMessage }
```
Comment (tachyons) [inline app/controllers/group_members_controller.rb]: - Do not use camelCase for variables in ruby, use snake_case
- No need of temporary variable here, just inline the message

Comment (ShubhankarKG) [reply app/controllers/group_members_controller.rb]: Codeclimate gave an error of line length being too long. That's why I had to introduce a variable.

Comment (ShubhankarKG) [reply app/controllers/group_members_controller.rb]: Ill update accordingly.

File: app/controllers/group_members_controller.rb [hunk 2]
```diff
@@ -90,10 +90,9 @@ def create
   # DELETE /group_members/1
   # DELETE /group_members/1.json
   def destroy
-
     @group_member.destroy
     respond_to do |format|
-      format.html { redirect_to group_path(@group_member.group), notice: 'Group member was successfully destroyed.' }
+      format.html { redirect_to group_path(@group_member.group), notice: "Group member was successfully removed." }
```
[the remark this answers is not in the document]
Comment (ShubhankarKG) [reply app/controllers/group_members_controller.rb]: As I mentioned previously, @tachyons .

[the remark this answers is not in the document]
Comment (tachyons) [reply app/controllers/group_members_controller.rb]: You can split this into two lines

**Comment by ShubhankarKG** [comment]
Please have a look now @tachyons .

**Comment by satu0king** [comment]
> Please have a look now @tachyons .

@ShubhankarKG, as @tachyons suggested, no need to make a temporary variable to store the message, keep the message inline. Split the line into two lines if necessary.

**Comment by ShubhankarKG** [comment]
Any idea how to fix coverage issues ?

**Comment by tachyons** [comment]
Remove unrelated changes

**Comment by ShubhankarKG** [comment]
This branch has been clogged due to commits being referenced while I was commiting. As a result, I'm pushing the changes to a new Branch. @tachyons If not needed, please close this PR.

**Pull request closed without merging**